src/opnsense/mvc/app/controllers/OPNsense/Dnsmasq/Api/SettingsController.php

```diff
@@ -36,6 +36,24 @@ class SettingsController extends ApiMutableModelControllerBase
     protected static $internalModelName = 'dnsmasq';
     protected static $internalModelClass = '\OPNsense\Dnsmasq\Dnsmasq';
 
+    /**
+     * Create a lookup map of tag UUIDs to tag names
+     *
+     * @return array uuid => name
+     */
+    private function getTagUuidMap(): array
```

Comment:
for simplicity I would only create a single method to collect options, a tag or an interface are technically the same anyway (it's either a `uuid` or an interface name)